src/scanner.c

```diff
@@ -207,36 +207,37 @@ static void GetIdent(void)
            so we recurse instead                                           */
         if ( c == '\\' ) {
             c = GET_NEXT_CHAR();
-            if ( c == 'n'  && i < SAFE_VALUE_SIZE-1 )  STATE(Value)[i] = '\n';
-            else if ( c == 't'  && i < SAFE_VALUE_SIZE-1 )  STATE(Value)[i] = '\t';
-            else if ( c == 'r'  && i < SAFE_VALUE_SIZE-1 )  STATE(Value)[i] = '\r';
-            else if ( c == 'b'  && i < SAFE_VALUE_SIZE-1 )  STATE(Value)[i] = '\b';
-            else if ( i < SAFE_VALUE_SIZE-1 )  {
-                STATE(Value)[i] = c;
+            switch(c) {
+            case 'n': c = '\n'; break;
+            case 't': c = '\t'; break;
+            case 'r': c = '\r'; break;
+            case 'b': c = '\b'; break;
+            default:
```

Comment:
Just checking you are aware you've changed the semantisc here (now isQuoted=1 is true even if i < SAFE_VALUE_SIZE-1)

Reply:
I suppose you mean `i >= SAFE_VALUE_SIZE-1`.  Yes, I am aware: while this change the semantics locally, it shouldn't matter globally, because if `i >= SAFE_VALUE_SIZE-1` then we trigger a syntax error anyway.

This reminds me: I should continue my PR for consistently enforcing a single length restriction for identifiers (IIRC, currently, some places use 1022, some 1023, some 1024 and some none).